index.js

```diff
@@ -88,7 +118,14 @@ console.log('hello world')
  * Example Output: "I called an arrow function!"
  */
 
-// TODO: ADD CODE HERE
+myArrowFuction = () => {
```

Comment:
When you define an arrow function, the variable name it's stored in needs a keyword:

```js
const myArrowFunction = () => {}
```